### DISCUSSION TITLE
suggestions in the github issues

### DISCUSSION THREAD

**Discussion by dawnch0rus** [ntsc-rs/ntsc-rs#70] [General]
anyone else think it might be best to redirect all the suggestions from the issues to discussions (here)? it doesn't seem like a great idea in the long run to have random suggestions in the same place as legitimate bugs and issues

**Comment by valadaptive** [comment]
I think it's useful to be able to track both suggestions and issues--there are many other projects that use the issue tracker in the same way. The "enhancement" and "bug" tags can be used to differentiate between the two. It is a bit annoying that GH's issue tracker is still so primitive, but I do want genuinely useful suggestions to be trackable.
